build/Targets/VSL.Settings.targets

```diff
@@ -102,7 +102,7 @@
             Condition="Exists('$(RoslynDiagnosticsPropsFilePath)') And ('$(UseRoslynAnalyzers)' == 'true')" />
   </ImportGroup>
 
-  <Target Name="EnsureGlobalNuGets" Condition="'$(BootstrapBuildPath)' == ''">
+  <Target Name="EnsureGlobalNuGets" Condition="'$(BootstrapBuildPath)' == '' AND '$(BuildingProject)' == 'true'">
```

Comment:
Should this be '$(BuildingProject)' != 'false'? I don't know if or when we care about points when it's unset.